src/api/baseUrls.js

```diff
@@ -0,0 +1,5 @@
+const BaseUrls = {
+    YOUTUBE: 'https://www.googleapis.com/youtube/v3',
+};
+
+export default BaseUrls;
```

Comment:
If you have default import and your file only consists of one single variable, please consider the following signature:
```suggestion
export default {
    YOUTUBE: 'https://www.googleapis.com/youtube/v3',
};
```